src/course/syllabus/application/tribute-page/resources.md

```diff
@@ -0,0 +1,35 @@
+## Getting started
+
+- [Setting up a web page](https://oliverjam.es/blog/complete-guide-to-making-web-pages/)
+- [freeCodeCamp: Responsive web design](https://www.freecodecamp.org/learn/2022/responsive-web-design/)
+  > Note: sections of freeCodeCamp's curriculum used to be a part of our application requirements. We replaced it with our own projects. Around the same time, freeCodeCamp published a new project-based Responsive Web Design curriculum. It doesn't touch much on JavaScript and completing it all would take you a reasonable while, it will give you a fantastic foundation in building websites.
+
+## HTML
+
+- [freeCodeCamp: Basic HTML and HTML5](https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/)
+- [W3 Schools: HTML Tutorial](https://www.w3schools.com/html/default.asp)
+
+## CSS
+
+- [freeCodeCamp: Basic CSS](https://www.freecodecamp.org/learn/responsive-web-design/basic-css/)
+- [freeCodeCamp: Responsive Web Design Principles](https://www.freecodecamp.org/learn/responsive-web-design/responsive-web-design-principles/)
+- [WebAIM: Alt Text](https://webaim.org/techniques/alttext/)
+- [CSS box model](https://www.w3schools.com/css/css_boxmodel.asp)
+
+## JavaScript
+
+- [Oli's intro to the DOM](https://oliverjam.es/blog/dom-intro/)
+- [MDN: Intro to DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction)
+- [Date.toLocaleString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString)
+
+## Git
+
+- [Git](https://git-scm.com/)
+- [GitHub's guide to set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+
+## GitHub
+
+- [Get started](https://docs.github.com/en/get-started)
+- [Create a repo](https://docs.github.com/en/get-started/quickstart/create-a-repo)
+- [Getting started with GitHub Pages](https://docs.github.com/en/pages/getting-started-with-github-pages)
+- [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow)
```

Comment:
we should be prescriptive on how we want people to commit. GitHub website or GitHub GUI app seem the most appropriate at this level.